src/v3/src/components/InputText/InputText.tsx

```diff
@@ -39,35 +45,50 @@ const InputText: UISchemaElementComponent<{
       messages = {},
       name,
     },
+    hint,
```

Comment:
what is this field for?

Reply:
This is the subLabel (text below the primary label), I think I should change the name to `subLabel`, originally I used `hint` because ODY team referred to it as that, but it is really just a sublabel.

Reply:
looks like there is data-se field be named as `subLabel`, we can try to keep consistency in our codebase first.

Reply:
Where are you seeing a data-se as subLabel? This sublabel in current v2 widget does not have a data-se prop, so maybe you are looking at something different?

Reply:
Oh you mean, I named it as SubLabel, yes, I will change this name.